web/js/codeworld_shared.js

```diff
@@ -1163,6 +1163,10 @@ function printMessage(type, message) {
     if (message.trim() === '') {
         return;
     }
+    // Deferred errors already handled on server side
+    if (message.indexOf("deferred") != -1){
+        return;
```

Comment:
Okay, I'm convinced that ignoring the error is good for now.  In the future, I'd like to add some logic where if the program has been running for a while, we add the "Giving up..." message we'd discussed.  But I agree that not including that is the better default, because 99% of the time, the program will crash during the first frame.